src/config/dynamic-theme-fixes.config

```
@@ -9494,6 +9494,13 @@ INVERT
 
 ================================
 
+radareu.cz
+
+INVERT
+#radarMap
```

```suggestion
#radarMap .leaflet-map-pane
```

These div only contains the panels etc and shouldn't revert all the other UI elements.